src/main.rs

```diff
@@ -39,10 +41,12 @@ fn main() -> rustyline::Result<()> {
     // Get args
     let args: Vec<String> = env::args().collect();
 
+    let mut llm_output = String::new();
+
     // Run aishrc file if it exists
     let aishrc = aishrc_path()?;
     if aishrc.exists() {
-        match run_file_mode(&aishrc) {
+        match run_file_mode(&aishrc, &mut llm_output) {
```

Comment:
How would the llm suggestions work with a file?

Reply:
Removed as not required